### ISSUE TITLE
Rotating a storage monitor does not rotate the item inside

### ISSUE THREAD

**Issue by ramidzkh** [AppliedEnergistics/Applied-Energistics-2#5934]
### Describe the bug

Rotating a storage monitor does not rotate the item inside

### How to reproduce the bug

1. Place a storage monitor
2. Power the storage monitor
3. Place an item inside it
4. Lock it
5. Rotate it with a wrench

### Expected behavior

The item inside will be rotated too

### Which minecraft version are you using?

1.18

### On which mod loaders does it happen?

Fabric

### Crash log

https://gist.github.com/ramidzkh/1c1fb3f2dd435bc6525807b688353f64

**Comment by shartte** [comment]
It should rotate when on the ground/ceiling. Should it rotate when on a wall? What happens to the amount???

**Comment by ramidzkh** [comment]
It rotates on the floor or ceiling. The item nor the amount rotates when places on a wall though

**Comment by shartte** [comment]
Yeah, I am not sure it should. Do people want this? What use is an upside down amount?

**Comment by Ommina** [comment]
The OP says "The item inside will be rotated too" -- the "too" kind of implies (to me anyway) that the model is rotating, while the display isn't.

If that is the case, it seems to me that the model (and its contents!) should just stay put.

_That said_!  I suppose it might be kind of interesting to keep the contents displayed correctly, but rotate the position of the quantity.

**Comment by ramidzkh** [comment]
There's no point rotating just the model of the monitor as it's mostly symmetrical anyway (just shades change iirc)
